medications/README.md

```diff
@@ -0,0 +1,152 @@
+| component / ehrSupplyPrescribe / code \[@displayName]                                    | 1. If `MedicationStatement.extension[0].valueCodeableConcept.coding[0].code` is `"prescribed-at-gp-practice"` OR `"prescribed-by-previous-practice"` then value is `NHS Prescription`.<br/>2. If `MedicationStatement.extension[0].valueCodeableConcept.coding[0].code` is `"prescribed-by-another-organisation"` then value is `Prescription by another organisation`. |
+| component / ehrSupplyPrescribe / code \[@codeSystem]                                     | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                                                                                                                                                                                                                                                                        |
+| component / ehrSupplyPrescribe / statusCode \[@code]                                     | Mapped from `MedicationRequest.status` - either `ACTIVE` or `COMPLETE`                                                                                                                                                                                                                                                                                                  |
+| component / ehrSupplyPrescribe / availabilityTime \[@value]                              | Mapped from `MedicationRequest.dispenseRequest.validityPeriod.start`                                                                                                                                                                                                                                                                                                    |
+| component / ehrSupplyPrescribe / quantity \[@value]                                      | If present then: `MedicationRequest.dispenseRequest.quantity.value` otherwise defaults to `"1"`                                                                                                                                                                                                                                                                         |
+| component / ehrSupplyPrescribe / quantity \[@unit]                                       | If present then: `MedicationRequest.dispenseRequest.quantity.value`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.extension`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.quantity.extension`.<br/>If none present then default value is `"Unk UoM"` (unknown unit of measurement).                                          |
```

Comment:
should this be `MedicationRequest.dispenseRequest.quantity.unit`?

Reply:
also, do we need to say if the extension has a url of https://fhir.nhs.`uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationQuantityText-1`?